src/Page.re

```diff
@@ -95,8 +99,8 @@ external setExtraHTTPHeaders : (~headers: Js.Dict.t(string), unit) => Js.Promise
 
 type typeOptions = {. "delay": float};
 
-[@bs.send.pipe : t]
```

Comment:
Let's keep it using `bs.send.pipe` for now, at least until there's an object first pipe syntax available. Same for `setContent`, and `Page.type_`, though it doesn't really matter for `content`.

Reply:
Fixed in commit 1c1183a, 3d64908 and c8c07aa.